### ISSUE TITLE
add current cmt picturers

### ISSUE THREAD

**Issue by butlerx** [redbrick/static-site#149]
https://www.redbrick.dcu.ie/about/committee/

**Comment by VoyTechnology** [comment]
You can use mine from gh

**Comment by theycallmemac** [comment]
also use my gh pic

**Comment by chrismarydowling** [comment]
Use my facebook profile picture

**Comment by butlerx** [comment]
the helpdesks also need there photos added too https://www.redbrick.dcu.ie/help/

**Comment by VoyTechnology** [comment]
How about using images from gravatar?

**Comment by butlerx** [comment]
you still have the issue of cmt having to create them and add the gravatar link

**Comment by VoyTechnology** [comment]
Yup, you still need to create them, but it you want to change them some time you change it for multiple sites at once. A good compromise would be to default to gravatar and keep a local copy of images if someone doesn't have a profile, or other way around

**Comment by butlerx** [comment]
but the cmt would have to supply them, your current issue is no one giving you photos.

also when i made it i actually chose it this way because i didn't want them to be replaced or updated i wanted them to be how they were when they were on cmt
